src/resonate/scheduler/scheduler.py

```diff
@@ -513,10 +526,11 @@ def _get_data_from_rfi(self, rfi: RFI) -> dict[str, Any]:
                 "args": rfi.unit.args,
                 "kwargs": rfi.unit.kwargs,
             }
+            tags = {"resonate:invoke": rfi.opts.send_to or "default"}
+            headers = None
         else:
             assert_never(rfi.unit)
-
-        return data
+        return (data, tags, headers)
```

Comment:
```suggestion
        return (data, headers, tags)
```

I'm being super anal here but data and headers belong together so it'd be nice to have them in order